src/components/sidebars/ProcessSidebar.jsx

```diff
@@ -0,0 +1,65 @@
+import {
+  Box,
+  Button,
+  Paragraph,
+  SectionHeading,
+  List,
+  ListItem,
+  Badge,
+} from "@contentful/f36-components";
+import { setScreen } from "../../state/screenSlice";
+import { SCHEDULE } from "../../constants/screens";
+import { useDispatch, useSelector } from "react-redux";
+import React from "react";
+import { setAppStatus } from "../../state/appStatusSlice";
+import * as AppStatus from "../../constants/app-status";
+import {
+  getMatchedSuppliersInContentful,
+  getSuppliersNotInContentful,
+} from "../../selectors";
+
+const ProcessSidebar = () => {
+  const dispatch = useDispatch();
+
+  const suppliersInContentful = useSelector(getMatchedSuppliersInContentful);
+  const suppliersNotInContentful = useSelector(getSuppliersNotInContentful);
+
+  const clickHandler = () => {
+    dispatch(setAppStatus(AppStatus.PROCESSING_SUPPLIERS));
+    dispatch(setScreen(SCHEDULE));
+  };
+
+  return (
+    <React.Fragment>
+      <SectionHeading>PROCESS SUPPLIERS</SectionHeading>
+      <Box marginTop="spacingM">
+        <Paragraph marginBottom="spacingM">When you click 'Process':</Paragraph>
```

Comment:
Small nitpick - the button below is 'Process suppliers' rather than just 'Process'